pybikes/bonopark.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016, Ben Caller <bcaller@gmail.com>
+# Distributed under the LGPL license, see LICENSE.txt
+
+import json
+import hashlib
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+__all__ = ['Bonopark', 'BonoparkStation']
+
+SECRET = 'APP_INICIO'
+BODY_DICT = {
+    'dni': SECRET,
+    'id_auth': SECRET,
+    'id_security': hashlib.md5(SECRET * 2).hexdigest() + hashlib.md5(SECRET).hexdigest()
+}
+
+COLORS = ['green', 'red', 'yellow', 'gray']
+
+class Bonopark(BikeShareSystem):
+    sync = True
+    meta = {
+        'system': 'Bonopark',
+        'company': 'Bonopark'
+    }
+
+    def __init__(self, tag, meta, url):
+        super(Bonopark, self).__init__(tag, meta)
+        self.feed_url = url
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+        scraper.headers['Content-Type'] = 'application/json'
```

Comment:
Can we set this user agent  `Apache-HttpClient/UNAVAILABLE (java 1.4)`? Seems to be the one the app is using.